### DISCUSSION TITLE
Error: axios module not found

### DISCUSSION THREAD

**Discussion by ciarantr** [parcel-bundler/parcel#8214] [Q&A]
Hi, I am having an issue with the axios package.

The application is running node and is using pnpm package manager.

When the server is running I receive the following error: 

`index.js:61 Uncaught Error: Cannot find module 'axios'
    at newRequire (index.js:61:19)
    at newRequire (index.js:45:18)
    at localRequire (index.js:84:35)
    at kupI4.@parcel/transformer-js/src/esmodule-helpers.js (login.js:5:15)
    at newRequire (index.js:71:24)
    at localRequire (index.js:84:35)
    at 6eWNN../mapbox (index.js:3:1)
    at newRequire (index.js:71:24)
    at index.js:122:5
    at index.js:145:3`

I have removed my node modules folder and reinstalled, but the error still persists.

Any suggestions would be greatly appreciated.

Node version 16.5.1
Parcel : 2.6.0
axios: 0.27.2

**Comment by mischnic** [comment]
What does your package.json look like? I think axios is not actually included in the bundle.

**Reply by ciarantr** [reply]
Hi, I have it included in package.json file.

**Reply by ciarantr** [reply]
{
  "name": "natours",
  "version": "1.0.0",
  "description": "learning node, express and mongoDB",
  "main": "app.js",
  "engines": {
    "node": ">=14.0.0"
  },
  "targets": {
    "main": false,
    "default": {
      "distDir": "./public/js/bundle"
    }
  },
  "scripts": {
    "start:dev": "nodemon ./server.js",
    "start:prod": "NODE_ENV=production nodemon server.js",
    "debug": "ndb node.js",
    "watch:js": "parcel watch ./public/js/index.js",
    "build:js": "parcel build ./public/js/index.js --dist-dir ./public/js/bundle.js"
  },
  "keywords": [
    "node.js",
    "express",
    "mongodb"
  ],
  "author": "Ciaran Toner",
  "license": "ISC",
  "dependencies": {
    "axios": "^0.27.2",
    "bcryptjs": "^2.4.3",
    "cookie-parser": "^1.4.6",
    "dotenv": "^16.0.1",
    "express": "^4.18.1",
    "express-mongo-sanitize": "^2.2.0",
    "express-rate-limit": "^6.4.0",
    "helmet": "^5.1.0",
    "hpp": "^0.2.3",
    "html-to-text": "^8.2.0",
    "jsonwebtoken": "^8.5.1",
    "mongoose": "^6.3.6",
    "morgan": "^1.10.0",
    "multer": "1.4.5-lts.1",
    "nodemailer": "^6.7.5",
    "pug": "^3.0.2",
    "sharp": "^0.30.6",
    "slugify": "^1.6.5",
    "stripe": "^9.8.0",
    "validator": "^13.7.0",
    "xss-clean": "^0.1.1"
  },
  "devDependencies": {
    "eslint": "^8.17.0",
    "eslint-config-airbnb": "^19.0.4",
    "eslint-config-prettier": "^8.5.0",
    "eslint-plugin-import": "^2.26.0",
    "eslint-plugin-jsx-a11y": "^6.5.1",
    "eslint-plugin-node": "^11.1.0",
    "eslint-plugin-prettier": "^4.0.0",
    "eslint-plugin-react": "^7.30.0",
    "eslint-plugin-react-hooks": "^4.5.0",
    "nodemon": "^2.0.16",
    "parcel": "^2.6.0",
    "prettier": "^2.6.2"
  }
}
